release-lib-master.Jenkinsfile

```diff
@@ -0,0 +1,189 @@
+        labels:
+          app: "jenkins-${env.JOB_BASE_NAME}"
+          indy-pipeline-build-number: "${env.BUILD_NUMBER}"
+      spec:
+        containers:
+        - name: jnlp
+          image: quay.io/kaine/indy-stress-tester:latest
+          imagePullPolicy: Always
+          tty: true
+          env:
+          - name: USER
+            value: 'jenkins-k8s-config'
+          - name: HOME
+            value: /home/jenkins
+          resources:
+            requests:
+              memory: 1Gi
+              cpu: 1000m
+            limits:
+              memory: 1Gi
+              cpu: 1000m
+          volumeMounts:
+          - mountPath: /home/jenkins/sonatype
+            name: volume-0
+          - mountPath: /home/jenkins/gnupg_keys
+            name: volume-1
+          - mountPath: /mnt/ocp
+            name: volume-2
+          workingDir: /home/jenkins
+        volumes:
+        - name: volume-0
+          secret:
+            defaultMode: 420
+            secretName: sonatype-secrets
+        - name: volume-1
+          configMap:
+            defaultMode: 420
+            name: gnupg
+        - name: volume-2
+          configMap:
+            defaultMode: 420
+            name: jenkins-openshift-mappings
+      """
+    }
+  }
+  options {
+    //timestamps()
+    timeout(time: 120, unit: 'MINUTES')
+  }
+  environment {
+    PIPELINE_NAMESPACE = readFile('/run/secrets/kubernetes.io/serviceaccount/namespace').trim()
+    PIPELINE_USERNAME = sh(returnStdout: true, script: 'id -un').trim()
+  }
+  stages {
+    stage('git checkout') {
+      steps{
+        script{
+          checkout([$class      : 'GitSCM', branches: [[name: params.LIB_GIT_BRANCH]], doGenerateSubmoduleConfigurations: false,
+                    extensions  : [[$class: 'RelativeTargetDirectory', relativeTargetDir: params.LIB_NAME]], submoduleCfg: [],
+                    userRemoteConfigs: [[url: params.LIB_GIT_REPO]]])
+
+          echo "Prepare the release of ${params.LIB_GIT_REPO} branch: ${params.LIB_GIT_BRANCH}"
+
+          sh """#!/bin/bash
+          cd ${params.LIB_NAME}
+          git checkout ${params.LIB_GIT_BRANCH}
+          gpg --allow-secret-key-import --import /home/jenkins/gnupg_keys/private_key.txt
+          """
+        }
+      }
+    }
+    stage('Setup and Formating'){
+      steps{
+        script{
+          withCredentials([
+            usernamePassword(credentialsId:'GitHub-Bot', passwordVariable:'BOT_PASSWORD', usernameVariable:'BOT_USERNAME'),
+            usernamePassword(credentialsId:'OSS-Nexus-Bot', passwordVariable:'OSS_BOT_PASSWORD', usernameVariable:'OSS_BOT_USERNAME'),
+            string(credentialsId: 'gnupg_passphrase', variable: 'PASSPHRASE')
+          ]) {
+            dir(params.LIB_NAME){
+              sh """
+              git config --global user.email "${params.BOT_EMAIL}"
+              git config --global user.name "${BOT_USERNAME}"
+              mvn -B -s ../settings.xml -Pformatting clean install
```

Comment:
I think we might need `mvn -B -s ../settings.xml -Pformatting,cq install` here instead. I think the profile name has changed to `cq` in more recent parent pom versions.